joplin/js/editor.js

```diff
@@ -387,4 +396,36 @@ $(function() {
   $('#sections-count').each(function(index, element) {
     trackChange(element);
   });
+
+  // autocomplete for places
+  (function() {
+    var latlng = {
+      lat: 30.26,
+      lng: -97.73,
+    };
+    var placesAutocomplete = places({
+      appId: 'plHX3G9C5GRN',
+      apiKey: 'd9275f5f90784eb5ba59ff8ec581d9bb',
```

Comment:
should we hide these keys?

Reply:
we could, or import them as an env variable. but a)it was something slapped together as a quick POC, I haven't confirmed that we even want to use this particular service and b) im pretty sure the key would be exposed client side either way, i dont think its a real high priority at the moment but when we refactor we could address it